src/Altinn.Notifications.Email.Core/Enums/EmailContentType.cs

```diff
@@ -1,4 +1,4 @@
-﻿namespace Altinn.Notifications.Email.Core.Models;
+﻿namespace Altinn.Notifications.Email.Core.Enums;
```

Comment:
Namespaces should ideally be about the features and not the code. I've never been any strict in my reviews and I too use names like Model in namespaces, but I do think it is something we should avoid. Models, Enums and Interfaces should ideally be located with the related features. The alternative is to create a namespaces for classes called Classes. ;)

I know it's dumb to bring it up in a random PR, but it's something I would like us to improve on going forward. No immediate action required.

Reply:
I would still like to folder structure to easier navigate the code. But we can go away from the principle of the name spaces matching the folder structure